client-src/elements/chromedash-add-stage-dialog.js

```diff
@@ -90,17 +91,22 @@ class ChromedashAddStageDialog extends LitElement {
   }
 
   renderStageSelect() {
+    const offerChoice = CREATEABLE_STAGES[this.featureType].length > 1;
+    const initialValue = offerChoice ? 0 : CREATEABLE_STAGES[this.featureType][0];
+    this.canSubmit = !offerChoice;
```

Comment:
Will the user be able to submit the stage creation if there is only 1 stage choice?

Reply:
Yes they will be that is why I set the "this.canSubmit"